doc/conseil.sql

```diff
@@ -264,6 +264,7 @@ CREATE TABLE public.operations (
     consumed_gas numeric,
     storage_size numeric,
     paid_storage_size_diff numeric,
+    origination_contracts character varying,
```

Comment:
Could we rename this to `originated_contracts` please?

Reply:
Sure, I just followed the indications as written in the issue ticket 😄